types/dagre/index.d.ts

```diff
@@ -12,15 +12,27 @@ export namespace graphlib {
         nodes(): string[];
         node(id: any): any;
         setDefaultEdgeLabel(callback: () => void): Graph;
-        setEdge(sourceId: string, targetId: string, options?: { [key: string]: any }): Graph;
+        setDefaultNodeLabel(callback: () => void): Graph;
+        setEdge(sourceId: string, targetId: string, options?: { [key: string]: any }, value?: string): Graph;
         setGraph(options: { [key: string]: any }): Graph;
         setNode(id: string, node: { [key: string]: any }): Graph;
+        graph(): GraphOptions;
+
+        constructor(opt?: {});
```

Comment:
Do you intend to fill out the [options this constructor accepts](https://github.com/cpettitt/dagre/blob/master/dist/dagre.js#L3510)?